### DISCUSSION TITLE
[bug] In linux `appWindow.isFocused` is always `false`.

### DISCUSSION THREAD

**Discussion by ayangweb** [tauri-apps/tauri#11344] [Q&A]
Hey👋, `appWindow.isFocused` is always `false` in linux, but it's fine in macOS and Windows! How can this problem be solved?

```ts
const appWindow = getCurrentWebviewWindow();

const focused = await appWindow.isFocused();
```

**Comment by FabianLars** [comment]
Duplicate of https://github.com/tauri-apps/tauri/issues/11323
